core/src/test/java/org/opensearch/sql/expression/aggregation/CountAggregatorTest.java

```diff
@@ -129,6 +129,21 @@ public void filtered_count() {
     assertEquals(3, result.value());
   }
 
+  @Test
+  public void distinct_count() {
```

Comment:
could you also test the complex type? e.g. list and map

Reply:
Done, thanks!